docs/_ext/powershell_lexer.py

```diff
@@ -0,0 +1,5 @@
+from pygments.lexers import get_lexer_by_name  # refer LEXERS
+
+
+def setup(app):
+    app.add_lexer('PowerShell', get_lexer_by_name('powershell'))
```

Comment:
please make this lower case - in keeping with all the other lexers

Reply:
fixed in 96aa0b05d9f83941cd1a072eeeaa3d5badc77997